### PR TITLE
Only build master and v\d.*

### PR DESCRIPTION
**Pull request by altendky**

### CODE REVIEW

**Comment by altendky** [comment]
I specifically pushed to `python-desert` rather than `altendky` to test.  It didn't seem to trigger a build and there's only the two (.org and .com) builds for the PR.  A follow-up empty test commit also triggered only those two builds.  It has not been tested against a tag, in case you want to try that out.

I picked a relatively broad regex because I would rather get a build for a v8 branch than miss a build for a release version.

**Review by ghost — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 language: python
 os: linux
+branches:
+  only:
+    - master
+    - /^v\d.*$/
```
Comment (python-desert) [inline .travis.yml:8]: Needs a comment explaining the conditions under which CI should run.

**Review by ghost — CHANGES_REQUESTED** [review]
Needs a comment

**Comment by altendky** [comment]
Kick CI to check #50

**Review by ghost — APPROVED** [review]

**Pull request merged**